package com.Exception;

public class Finallyprogram {
	public static void main(String args[])
	{
		try
		{
			System.out.println("join job");
			System.out.println("attend class reguraly");
		int a=1/0;
		System.out.println("attend daily");
		System.out.println("proper writing programs");
		}
		catch(ArithmeticException e)
		{
			System.out.println("attend back up classes");	
		}
		finally
		{
			System.out.println("get good opinion on company");
			System.out.println("be happy");
		}
	}
}
